dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -348,7 +348,7 @@ docs.prow:
 # https://github.com/kubernetes-sigs/cluster-api docs (@dwat, @jdetiber, @justinsb)
 cluster-api.sigs:
   type: CNAME
-  value: release-1-3--kubernetes-sigs-cluster-api.netlify.app.
+  value: release-1-4--kubernetes-sigs-cluster-api.netlify.app.
```

Comment:
Shouldn't this still be release-1-3 till v1.4.0 is released?

Reply:
I also thought about this when looking at the [reference](https://github.com/kubernetes/k8s.io/pull/3872/files). But I assumed this would just be held until v1.4 was almost ready. But I can remove this part if preferred.

Reply:
I'm not sure how it is relevant if we link the release-1.4 book anywhere or not.

This change should change https://cluster-api.sigs.k8s.io/ to point to release-1.4.

As far as I remember this doesn't actually happen because netlify does some magic. Only after we did the steps in "Change production branch in Netlify to the new release branch" "cluster-api.sigs.k8s.io" will actually host the 1.4 book.

(I think that's what we discovered one of the last few times)

Reply:
Let's merge this PR now to make sure the certificate thing works

Reply:
Sounds good. Lets merge this.